hil_extension/load-config.sh

```diff
@@ -0,0 +1,21 @@
+# For existing shell scripts to use the ../conf/quads.yml
```

Comment:
Can you comment as to why this has to be `source`d? And why it can't be just included in its parent script?